README.md

```diff
@@ -5,6 +5,9 @@
 ExMachina makes it easy to create test data and associations. It works great
 with Ecto, but is configurable to work with any persistence library.
 
+> **This README is for the unreleased beta version of ExMachina**. Here are the 
```

Comment:
How about something like:

This README follows master, which may not be the currently released version.

Not picky about the wording. My main concerns were to make it clear that sometimes the docs may be the same and also that this isn't necessarily a "beta" version since it's just master.